packages/ove-service-imagetiles/src/OVE.Service.ImageTiles/Startup.cs

```diff
@@ -91,38 +94,58 @@ public class Startup {
         /// <summary>
         /// Register this OVE service with the Asset Manager Service 
         /// </summary>
-        /// <param name="app">kestrel app</param>
-        private async void RegisterServiceWithAssetManager(IApplicationBuilder app) {
+        private async void RegisterServiceWithAssetManager() {
+
+            // get the service description from the AppSettings.json 
+
             OVEService service = new OVEService();
             Configuration.Bind("Service", service);
-            
-            service.ViewIFrameUrl = Configuration.GetValue<string>("ServiceHostUrl") + "/api/ImageController/ViewImage/?id={id}"; 
 
-            // update the real processing states
+            service.ViewIFrameUrl = Configuration.GetValue<string>("ServiceHostUrl") +
+                                    "/api/ImageController/ViewImage/?id={id}";
+
+            // then update the real processing states
             service.ProcessingStates.Clear();
             foreach (var state in Enum.GetValues(typeof(ProcessingStates))) {
                 service.ProcessingStates.Add(((int) state).ToString(), state.ToString());
             }
 
             // register the service
 
-            string url = Configuration.GetValue<string>("AssetManagerHostUrl") +
-                         Configuration.GetValue<string>("RegistrationApi");
-
-            Console.WriteLine("About to register with url " + url + " we are on " + service.ViewIFrameUrl);
-
-            using (var client = new HttpClient()) {
-                var responseMessage = await client.PostAsJsonAsync(url, service);
-
-                Console.WriteLine("Result of Registration was " + responseMessage.StatusCode);
+            bool registered = false;
+            while (!registered) {
+                string url = null;
+                try {
+                    // permit environmental variables to be updated 
+                    url = Configuration.GetValue<string>("AssetManagerHostUrl") +

```

Comment:
This will conflict with issue-29 branch. I would put this review on hold until everything else is merged